docs/user/dashboard/lens-end-to-end/lens-end-to-end.asciidoc

```diff
@@ -0,0 +1,506 @@
+[float]
+[[lens-end-to-end]]
+== Try it: Analyze website data on a dashboard
+
+You have collected data about your webiste, and you're ready to find the answers to some key questions. 
+
+When you're finished creating the dashboard, you'll be able to answer the following questions:
+
+* How many users have visited my website?
+* What’s the distribution of visitors by operating system?
+* What’s the ratio of code responses over time?
+* What’s the average bytes transfer per day? How does it relate to the number of visitors?
+* What's the percentage of small vs big files transfer?
```

Comment:
Now when I am reading it, I think maybe 'vs' might be not non-native-speaker friendly word. I'd replace it with 'to'